packages/block-editor/src/style.scss

```diff
@@ -1,3 +1,7 @@
+#start-resizable-editor-section {
```

Comment:
@jorgefilipecosta Can we use the simulated queries introduced here in the customizer sidebar?

Reply:
Yes, I think it would work as expected in the customizer view we would useSimulatedMediaQuery with a value that forces a mobile-like view. It will probably not work magically right away and we will probably need some adjustments to make things look well on the customizer like I did in https://github.com/WordPress/gutenberg/pull/17960/.